static/js/translation.js

```diff
@@ -0,0 +1,60 @@
+
+//load initial language file.
+languageStartup();
+
+function languageStartup(){
+    var url = window.location.href;
+    console.log(url);
+    var langChoice = url.split('?lang=')[1] || "en";
+    let langPath = "lang/" + langChoice + ".js";
```

Comment:
Does this assume there won't be any other parameters after "lang"?
Would the `?lang=en&somethingelse=false` URL try to load `lang/en&somethingelse=false.js`?

Reply:
Yep it assumes no other parameters are coming after, this is something I can change to future proof this action

Reply:
Yes please, as this would be affected by anything after the `#` as well, which will be attached to the URL quite easy (like pressing most of the buttons in the editor).